x-pack/filebeat/input/httpjson/value_tpl.go

```diff
@@ -71,6 +71,7 @@ func (t *valueTpl) Unpack(in string) error {
 			"mul":                 mul,
 			"now":                 now,
 			"parseDate":           parseDate,
+			"parseDateInTZ":       parseDateInTZ,
```

Comment:
This looks like it should be done in a separate PR...

Reply:
:laughing: Yeah, noodling while making this change and accidentally committed this.